src/sageworks/web_components/experiments/table_comparison.py

```diff
@@ -76,7 +76,14 @@ def update_properties(self, table_df: pd.DataFrame, **kwargs) -> list:
         table_data = table_df.to_dict("records")
 
         # Define column definitions based on the DataFrame
-        column_defs = [{"headerName": col, "field": col, "filter": "agTextColumnFilter"} for col in table_df.columns]
+        column_defs = [
```

Comment:
Obviously, all this hardcoded stuff will need to be removed.  The `update_properties` method should take a arg that specifies which columns are in 'markdown' (or link) mode.. the current code shows this...

```
 markdown_columns: list[str] = None) -> list:
      ...
        for c in show_columns:
            column_def = {
                "name": c,
                "id": c,
                "presentation": "markdown" if markdown_columns and c in markdown_columns else "input",
```

Reply:
yes it must be possible to do something more general to make it usable for any dataset.